### PR TITLE
Add react sample application.

### PR DESCRIPTION
**Pull request by AfraHussaindeen**
## Purpose

- $Subject

## Guardio Life sample application demo :
https://drive.google.com/file/d/16gFUjQTXsBmkzig2AMHEfLvvSjBHOuYB/view?usp=drive_link

### CODE REVIEW

File: samples/react-sample-app/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (pavinduLakshan) [inline samples/react-sample-app/webpack.config.js]:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

File: samples/react-sample-app/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
```
Comment (pavinduLakshan) [inline samples/react-sample-app/webpack.config.js]:
```suggestion
 * WSO2 LLC. licenses this file to you under the Apache License,
```

File: samples/react-sample-app/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+const envFile = ".env";
+const envPath = path.resolve(__dirname, envFile);
+const envVars = require('dotenv').config({ path: envPath }).parsed || {};
+
+const generatePortInUsePrompt = () => {
+    return `Be sure to update the following configurations if you proceed with the port change.
+
+    1. Update the "PORT" in ".env" file in the app root.
+    2. Update the signInRedirectURL & signOutRedirectURL in "src/runtime-config.json".
+    3. Go to the Identity server/Asgardeo console and navigate to the protocol tab of your application:
```
Comment (pavinduLakshan) [inline samples/react-sample-app/webpack.config.js]:
```suggestion
    3. Go to the WSO2 Identity server/Asgardeo console and navigate to the protocol tab of your application:
```

File: samples/react-sample-app/src/util/IdvProviderMessages.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                        color="text.secondary" 
+                        sx={{ mt: 2, mb: 4 }}
+                    >
+                        Please open &quot;public/runtime-config.json&quot; file using an editor, and add
+                        the <code>clientID</code> value with the registered application&apos;s client ID.
+                    </Typography>
+                    <Typography variant="body2" align="center">
+                        Visit the{' '}
+                        <Link 
+                            href="https://github.com/asgardeo/asgardeo-auth-react-sdk/tree/master/samples/asgardeo-react-app"
```
Comment (pavinduLakshan) [inline samples/react-sample-app/src/util/IdvProviderMessages.tsx]: Why do we point the end users to the readme of the sample app in react sdk here?

File: samples/react-sample-app/webpack.config.js (hunk 1 above)
Comment (AfraHussaindeen) [reply samples/react-sample-app/webpack.config.js]: Resolved via https://github.com/wso2-extensions/identity-verification-onfido/pull/17/commits/0781a5f177ea9d7bcfc911ee471148864738a222

File: samples/react-sample-app/webpack.config.js (hunk 3 above)
Comment (AfraHussaindeen) [reply samples/react-sample-app/webpack.config.js]: Resolved via https://github.com/wso2-extensions/identity-verification-onfido/pull/17/commits/0781a5f177ea9d7bcfc911ee471148864738a222

File: samples/react-sample-app/src/util/IdvProviderMessages.tsx (hunk 4 above)
Comment (AfraHussaindeen) [reply samples/react-sample-app/src/util/IdvProviderMessages.tsx]: Resolved via https://github.com/wso2-extensions/identity-verification-onfido/pull/17/commits/0781a5f177ea9d7bcfc911ee471148864738a222

File: samples/react-sample-app/webpack.config.js (hunk 2 above)
Comment (AfraHussaindeen) [reply samples/react-sample-app/webpack.config.js]: Resolved via https://github.com/wso2-extensions/identity-verification-onfido/pull/17/commits/0781a5f177ea9d7bcfc911ee471148864738a222

File: samples/react-sample-app/README.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+5. Once the verification is successfully completed, the plan selection buttons become active and clickable.
+6. Users can then proceed to select their desired insurance plan.
+
+This sample showcases the seamless integration of identity verification in a real-world scenario, enhancing security and compliance for online insurance purchases while providing a clear user experience with visual cues about the verification status.
+
+## Getting Started
+
+### Prerequisites
+- Ensure you have `Node.js` installed (version 10 or above).
+- Ensure that a user account is created, and the *first name*, *last name*, and *date of birth* are correctly set to match the details on the document that will be used for verification.
```
Comment (pavinduLakshan) [inline samples/react-sample-app/README.md:20]: Shall we mention the steps to create a user account?

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**